### DISCUSSION TITLE
Feature Request: Koa Server

### DISCUSSION THREAD

**Discussion by WilliamABradley** [ecyrbe/zodios#385] [Ideas]
This is a cool project!

It would be nice to have an official Koa offering, as Koa feels more modern, and handles promises naturally, which is pretty valuable in writing middlewares.

**Comment by ecyrbe** [comment]
Hello @WilliamABradley ,

Thank you for the suggestion.

Unfortunately zodios have not enough man power to be able to handle another library backend.

This can be a community effort or independant package if someone wants to tacle it.
